lib/app/layouts/conversation_view/widgets/header/header_widgets.dart

```diff
@@ -64,9 +65,9 @@ class ManualMarkState extends OptimizedState<ManualMark> {
                 marking = true;
               });
               if (!marked) {
-                await http.markChatRead(chat.guid);
+                await backend.markRead(chat);
               } else {
-                await http.markChatUnread(chat.guid);
+                await backend.getRemoteService()?.markChatUnread(chat.guid);
```

Comment:
Rust push cant mark unread yet? I feel like this markChatUnread should still be a global thing and not hidden behind the remoteService, as many backends support marking unread.

Reply:
Honestly completely forgot this syncs between devices lol